features/product_index.feature

```diff
@@ -0,0 +1,10 @@
+Feature: View the Index Page of Products
```

Comment:
The feature and step names should be clarified as to whether it's for the volunteer-facing or customer-facing index pages.